Source/engine/render/text_render.cpp

```diff
@@ -109,19 +133,17 @@ void LoadFont(GameFontTables size, text_color color)
 		LoadMaskedArt(path, font.get(), 256, 1);
 	}
 
-	uint32_t fontId = (color << 24) | (size << 16);
-	Fonts.insert(make_pair(fontId, move(font)));
-
-	sprintf(path, "fonts\\%i-00.bin", FontSizes[size]);
-	LoadFileInMem(path, FontKerns[size]);
+	return Fonts.insert(make_pair(fontId, move(font))).first->second.get();

```

Comment:
Same as with `FornKerns`: no need for the `unique_ptr`.

Reply:
wont we leak when unloading things?

Reply:
I guess so long as we don't store pointers to dynamically allocated memory in the `unordered_map`, then it shouldn't be a problem. In other words, we could start leaking if we used `std::unordered_map<uint32_t, Art*>` as opposed to `std::unordered_map<uint32_t, Art>`.